packages/jest-mock/src/index.ts

```diff
@@ -50,15 +50,13 @@ export type MaybeMockedConstructor<T> = T extends new (
 ) => infer R
   ? MockInstance<R, ConstructorArgumentsOf<T>>
   : T;
-export type MockedFunction<T extends MockableFunction> = MockWithArgs<T> & {
+export type MockedFn<T extends MockableFunction> = MockWithArgs<T> & {
```

Comment:
I see zero reason to use an abbreviation - what `@types/jest` does is not really relevant

Reply:
yeah there is no absolute reason for doing this , I thought that the things should looks consistent but if not that's the case i can revert this ? 🙂

Reply:
reverting this too